src/wasmopcodecnt.c

```diff
@@ -177,7 +177,8 @@ int main(int argc, char** argv) {
       result = WASM_ERROR;
     }
     if (WASM_SUCCEEDED(result)) {
-      for (size_t i = 0; i < WASM_ARRAY_SIZE(wasm_opcode_count); ++i) {
```

Comment:
I wasn't building in C99 mode, because it seemed like the Windows build wasn't able to. This is why I removed variable initialization with `= {};` and implemented hex parsing as well.

Perhaps we can switch to VS2015 which supports these features, but I'm hesitant without @Cellule chiming in, since he made most of those changes.